dotcom-rendering/scripts/test-data/gen-fixtures.js

```diff
@@ -351,6 +352,48 @@ requests.push(
 		}),
 );
 
+requests.push(
+	// ToDo: run vs production (currently generated vs localhost)
```

Comment:
What does this mean?

Reply:
Fixed in https://github.com/guardian/dotcom-rendering/pull/13733/commits/5d92c02cf4e5d847bc2a3aa1d325501fea02642d